static/app/utils/usePrefersReducedMotion.tsx

```diff
@@ -0,0 +1,5 @@
+import useMedia from 'sentry/utils/useMedia';
+
+export function usePrefersReducedMotion() {
+  return !useMedia('(prefers-reduced-motion: no-preference)');
+}
```

Comment:
Should we use https://www.framer.com/motion/guide-accessibility/#usereducedmotion

Reply:
Yup, didn't realize we were already using that